src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec

```diff
@@ -2,12 +2,16 @@
 <package>
  <metadata minClientVersion="3.3">
     <id>Microsoft.CodeAnalysis.Build.Tasks</id>
-    <summary>The build Tasks for running the C# and VB compilers in MSBuild.</summary>
+    <summary>
+      The build task and targets used by MSBuild to run the C# and VB compilers.
+    </summary>
     <description>
-        Contains the build task and targets used by MSBuild to run the C# and VB compilers.
-        Supports using VBCSCompiler on Windows.
+      The build task and targets used by MSBuild to run the C# and VB compilers.
+      Supports using VBCSCompiler on Windows.
```

Comment:
I must admit I'm not sure how a customer will understand what VBCSCompiler is, much less how this package supports it on Windows.

Reply:
I don't know either. I'll leave this one as-is.